src/components/jellyfinActions.ts

```diff
@@ -400,21 +400,14 @@ export async function detectBitrate(numBytes = 500000): Promise<number> {
 
 /**
  * Tell Jellyfin to kill off our active transcoding session
- * @param state - playback state.
+ * @param playSessionId - the play session ID to stop encoding
  * @returns Promise for the http request to go through
  */
-export function stopActiveEncodings(state: PlaybackState): Promise<void> {
-    const options = {
+export async function stopActiveEncodings(
+    playSessionId: string
+): Promise<void> {
+    await getHlsSegmentApi(JellyfinApi.jellyfinApi).stopEncodingProcess({
```

Comment:
I think you're missing an import

Reply:
I just didn't have more time to fix it during my lunch break :)